api.py

```diff
@@ -13,8 +13,10 @@ def __init__(self, client_id, client_secret):
         self.client_secret = client_secret
         self.token = None
 
+    """ Authenticates Client """
```

Comment:
Docstrings in python (these tripple quotes) should go inside the function/class. Also the return type should go inside the docstring like this:

```
def _auth_client(self):
""" Authenticate Client and return an access token."""
```

There are lots of ways to format python docstrings, but the important thing is to place them in the right place and put the required information in them.

When using an automatic documentation generator or some IDEs, they look at this docstring and show it as an explanation of the function.